src/ui/qgsrasterlayerpropertiesbase.ui

```diff
@@ -94,14 +94,32 @@
          </property>
          <item>
           <property name="text">
-           <string>General</string>
+           <string>Information</string>
+          </property>
+          <property name="icon">
+           <iconset resource="../../images/images.qrc">
+            <normaloff>:/images/themes/default/propertyicons/metadata.svg</normaloff>:/images/themes/default/propertyicons/metadata.svg</iconset>
+          </property>
+         </item>
+         <item>
+          <property name="text">
+           <string>Metadata</string>
           </property>
           <property name="toolTip">
            <string>General</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
-            <normaloff>:/images/themes/default/propertyicons/general.svg</normaloff>:/images/themes/default/propertyicons/general.svg</iconset>
+            <normaloff>:/images/themes/default/console/iconShowEditorConsole.png</normaloff>:/images/themes/default/console/iconShowEditorConsole.png</iconset>
```

Comment:
As of commit cef089a , you have a nice vector (and properly sized) image you should use instead of this PNG.